### PR TITLE
[4.0] Fix background shorthand incompatibilities

### PR DESCRIPTION
**Pull request by infograf768**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/27572
Replaces https://github.com/joomla/joomla-cms/pull/27798

### Summary of Changes
Separating background color and background image in CSS to allow loading the specific .svg in Safari and Edge.

### Testing Instructions
Use Safari or Windows Edge.
Log in administration area.

Go to Articles.
Click Filter Options.
Click Select Condition dropdown and select a condition.
Selected item has no down arrow.
Apply PR. 
USE NPM
Repeat steps above.
Selected items have down arrow.
Confirm with other browsers such as Firefox, Chrome work correctly as before.

Repeat for RTL.

### Before patch
27572-before

### After patch
27572-pr

UPDATED as it should now work in Safari

### CODE REVIEW

**Comment by infograf768** [comment]
@vistamedia @C-Lodder @ciar4n

File: administrator/templates/atum/scss/vendor/bootstrap/_custom-forms.scss [hunk 1]
```diff
@@ -2,11 +2,13 @@
 
 .custom-select {
   max-width: $input-max-width;
+  background: $custom-select-background;
```
Comment (Quy) [inline administrator/templates/atum/scss/vendor/bootstrap/_custom-forms.scss:5]: Will it work to split `background` to their individual properties and not use shorthand `background`?

Comment (infograf768) [reply administrator/templates/atum/scss/vendor/bootstrap/_custom-forms.scss:5]: That part concerns LTR and it does work fine for all browsers including Safari.
The problem is for RTL, and only in Safari. As I explain above.

**Comment by infograf768** [comment]
@Quy 
Please test RTL now both in Safari and other browsers
(Make sure cache is deleted for each browser)

NOTE: the `select-bg-rtl.svg` now contains the default background color used by Atum.
`fill="#1a466b"`
if this background color is modified in the future, the value in the svg should be modified to fit

**Comment by brianteeman** [comment]
I have tested this item :red_circle: unsuccessfully on e890de61273805066eaaa8352b54191645bc481e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27864">issues.joomla.org/tracker/joomla-cms/27864</a>.</sub>

**Comment by brianteeman** [comment]
When in RTL and with a custom hue selected in the admin template this doesnt work 100%

image

**Comment by infograf768** [comment]
> When in RTL and with a custom hue selected in the admin template this doesnt work 100%

Understand. If someone finds a solution, then please propose.

**Comment by infograf768** [comment]
Grrr. found out the issue...
My first try was minified which killed it.
Will update this pr tomorrow.

**Comment by infograf768** [comment]
@Quy @brianteeman 
Please test again. svg corrected.

**Comment by Quy** [comment]
Looks good except here. Edit an article. This is chosen that you mentioned requires fixing.

**Comment by infograf768** [comment]
That just means we have to add a dir rtl there. On it.

**Comment by infograf768** [comment]
@Quy
Fixed RTL choices for single choice

Screen Shot 2020-02-12 at 10 07 26

And as well RTL for multiple choices (It does not contain an svg but as I was at it, I did it)

Screen Shot 2020-02-12 at 10 08 51

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 08a40b330ff9dd824cff560a7a6f8d00dd9b6f52<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27864">issues.joomla.org/tracker/joomla-cms/27864</a>.</sub>

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 08a40b330ff9dd824cff560a7a6f8d00dd9b6f52<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27864">issues.joomla.org/tracker/joomla-cms/27864</a>.</sub>

**Comment by jwaisner** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27864">issues.joomla.org/tracker/joomla-cms/27864</a>.</sub>

**Pull request merged**